index.html

```diff
@@ -0,0 +1,201 @@
+                <input list="city_from" name="living_from" id="living_from">
+                <datalist id="city_from">
+                    <option value="Dublin">
+                    <option value="London">
+                    <option value="Amsterdam">
+                    <option value="Berlin">
+                    <option value="Kharkiv">
+                </datalist>
+                <img src="Images/logos/gray plane.png" alt="logo">
+                <label for="going_to">Going to</label>
+                <input list="city_to" name="going_to" id="going_to">
+                <datalist id="city_to">
+                    <option value="Dublin">
+                    <option value="London">
+                    <option value="Amsterdam">
+                    <option value="Berlin">
+                    <option value="Kharkiv">
+                </datalist>
+            </form>
+            <form class="dates" action="" method="get">
+                <label for="departure_date">Leave <img src="Images/logos/leave plane.png" alt="logo"></label>
+                <input type="date" id="departure_date" name="departure_date">
+                <img src="Images/logos/Planner.png" alt="logo">
+                <label for="return_date">Leave <img src="Images/logos/return plane.png" alt="logo"></label>
+                <input type="date" id="return_date" name="return_date">
+            </form>
+            <input class="search_button" type="submit" value="">
+        </div>
+    </div>
+    <main>
+        <h3 class="offers_header">Special Upcoming Offers</h3>
+        <div class="group_type">
+            <span>Team</span>
+            <span>Couple</span>
+            <span>Family</span>
+        </div>
+        <div class="special_offers">
+            <div class="offer_block">
+                <img src="Images/image/Loga sea.png" alt="Loga sea">
+                <p class="date">14 FEB 2022</p>
+                <p><span class="relax">Relax</span> 3 Days, 3 Nights</p>
+                <h4>Loga Sea</h4>
+                <p><b>700$</b>/Person</p>
+                <button>Book Now</button>
+            </div>
+            <div class="offer_block">
+                <img src="Images/image/Ansgar Scheffold.png" alt="Ansgar Scheffold">
+                <p class="date">18 JUN 2022</p>
+                <p><span class="adventure">Adventure</span> 4 Days, 3 Nights</p>
+                <h4>Ansgar Scheffold</h4>
+                <p><b>400$</b>/Person</p>
+                <button>Book Now</button>
+            </div>
+            <div class="offer_block">
+                <img src="Images/image/Lona X.png" alt="Lona X">
+                <p class="date">22 DEC 2022</p>
+                <p><span class="relax">Relax</span> 7 Days,6 Nights</p>
+                <h4>Lona X</h4>
+                <p><b>340$</b>/Person</p>
+                <button>Book Now</button>
+            </div>
+        </div>
+        <div class="about_us">
```

Comment:
When you logically separate content on the page use instead section tag. It's semantically closer to what you want to accomplish.

Reply:
I don't understand which unit you mean